lib/galaxy/webapps/galaxy/controllers/forms.py

```diff
@@ -303,28 +296,6 @@ def save_form_definition(self, trans, form_id=None, payload=None, **kwd):
             trans.sa_session.commit()
         return form_definition, None
 
-    @web.expose
-    @web.require_admin
-    def _delete_form(self, trans, ids):
-        for form_id in ids:
-            form = get_form(trans, form_id)
-            form.deleted = True
-            trans.sa_session.add(form)
-            with transaction(trans.sa_session):
-                trans.sa_session.commit()
-        return ("Deleted %i form(s)." % len(ids), "done")
-
-    @web.expose
-    @web.require_admin
-    def _undelete_form(self, trans, ids):
-        for form_id in ids:
-            form = get_form(trans, form_id)
-            form.deleted = False
-            trans.sa_session.add(form)
-            with transaction(trans.sa_session):
-                trans.sa_session.commit()
-        return ("Undeleted %i form(s)." % len(ids), "done")
-
 
```

Comment:
This *Utility methods* block can be removed and the `get_form` utility can be then imported from `galaxy.managers.forms`

Reply:
Thank you!